scripts/run-ci-e2e-tests.js

```diff
@@ -58,7 +58,11 @@ try {
   }
 
   describe('Create react-native package');
-  if (exec('node ./scripts/set-rn-version.js --version 1000.0.0').code) {
+  if (
+    exec(
+      'node ./scripts/set-rn-version.js --to-version 1000.0.0 --build-type dry-run',
+    ).code
+  ) {
```

Comment:
this change makes the CircleCI `test_js` and `test_prev_lts_js` fails miserably because they can't find ruby anymore.

It doesn't make any sense to me, unless there are different scripts involved.

Reply:
did you end up figuring out what was going on here? did rebasing on top of main help? 🤔

Reply:
no, unfortunately. No idea why this invocation make it fail ruby... :/